projects/auth0-angular/src/lib/auth.interceptor.ts

```diff
@@ -113,6 +113,13 @@ export class AuthHttpInterceptor implements HttpInterceptor {
     };
 
     if (isHttpInterceptorRouteConfig(route)) {
+      if (
+        route.httpMethod &&
+        route.httpMethod.toLowerCase() !== request.method.toLowerCase()
```

Comment:
I think we should be comparing against the upper case, without transforming if we want to fail when there's no match. According to this, it looks like method names are case sensitive https://tools.ietf.org/html/rfc7230#section-3.1.1 and https://tools.ietf.org/html/rfc7231#section-4. But we can sync and evaluate this, e.g. what other libraries do